lib/chef/knife.rb

```diff
@@ -510,8 +510,8 @@ def format_rest_error(response)
       response.body
     end
 
-    def create_object(object, pretty_name = nil, &block)
-      output = edit_data(object)
+    def create_object(object, pretty_name = nil, klass: nil, &block)
```

Comment:
I know `klass` is pretty standard but since the data for the object is called `object`, `object_class` might be clear and has the benefit of being made up of real words :)